tests/functional/metadata/mongodb/listObject.spec.js

```diff
@@ -477,6 +479,25 @@ describe('MongoClientInterface::metadata.listObject', () => {
                     }),
                 ], done);
             });
+
```

Comment:
can we add a test for the "error" case as well?

Reply:
Done in https://github.com/scality/Arsenal/pull/2177/commits/4aa8b5cc6e82d4f69773e5d3cedcc60b9a22f39c